app/src/main/res/layout/activity_registration.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
please run the project before commiting changes, you need to be sure your changes are running. On my device it looks like this:

![image](https://user-images.githubusercontent.com/88314300/178220251-6f3e8bbc-a699-4834-a0cd-afed40085683.png)